src/cmd/completion.go

```diff
@@ -0,0 +1,69 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var completionNoDesc bool
+
+var completionCmd = &cobra.Command{
+	Use:   "completion [bash|zsh|fish|powershell]",
+	Short: "Generate completion script",
+	Long: `To load completions:
+
+Bash:
+
+$ source <(git-town completion bash)
+
+# To load completions for each session, execute once:
+Linux:
+  $ git-town completion bash > /etc/bash_completion.d/git-town
+MacOS:
+  $ git-town completion bash > /usr/local/etc/bash_completion.d/git-town
+
+Zsh:
+
+$ source <(git-town completion zsh)
+
+# To load completions for each session, execute once:
+$ git-town completion zsh > /usr/share/zsh/vendor-completions/_git-town
+
+Fish:
+
+$ git-town completion fish | source
+
+# To load completions for each session, execute once:
+$ git-town completion fish > ~/.config/fish/completions/git-town.fish
```

Comment:
Home?

Reply:
I don't understand your comment. The path looks okay to me.

Reply:
for dotfile repos..